examples/gno.land/p/demo/uuid/uuid.gno

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 The godruoyi/gno-snowflake Author. All rights reserved.
+// Use of this source code is governed by a MIT.
+
+package uuid
+
+import (
+	"errors"
+	"time"
+)
+
+// These constants are the bit lengths of snowflake ID parts.
+const (
+	TimestampLength uint8  = 41
+	MachineIDLength uint8  = 10
+	SequenceLength  uint8  = 12
+	MaxSequence     uint16 = 1<<SequenceLength - 1
+	MaxTimestamp    uint64 = 1<<TimestampLength - 1
+	MaxMachineID    uint64 = 1<<MachineIDLength - 1
+
+	machineIDMoveLength = SequenceLength
+	timestampMoveLength = MachineIDLength + SequenceLength
+)
+
+// default start time is 2008-11-10 23:00:00 UTC, why ? In the playground the time begins at 2009-11-10 23:00:00 UTC.
+// It can run on golang playground.
+// default machineID is 0
+type SequenceResolver func(ms int64) (uint16, error)
+
+var (
+	resolver      SequenceResolver
+	MachineID     uint64 = 0
+	sequence      uint16 = 0
+	StartTime            = time.Date(2008, 11, 10, 23, 0, 0, 0, time.UTC)
+	LastTimestamp int64  = 0
+	// _             int    //will not sync atomic Gno not goroutine, implement in the future for the resolver.
+)
+
+// ID use ID to generate snowflake id, and it will ignore error. if you want error info, you need use NextID method.
+func ID() uint64 {
+	id, _ := NextID()
+	return id
+}
+
+// NextID use NextID to generate snowflake id and return an error.
+func NextID() (uint64, error) {
```

Comment:
I think we can have a UUID implementation that cannot fail, as we do not depend on external resources like the system clock.

I suggest having `ID() uint64` and removing `NextID() (uint64, error)`.

Reply:
[`2f1bc74`](https://github.com/gnolang/gno/pull/2076/commits/2f1bc74778a2dd8937f8bcbe5c51ee087edbf249)